frontstage/views/secure_messaging/create_message.py

```diff
@@ -77,3 +77,26 @@ def send_message(party_id, is_draft, case_id, survey, ru_ref):
     logger.info('Secure message sent successfully',
                 message_id=sent_message['msg_id'], party_id=party_id)
     return sent_message
+
+
+def create_headers():
+    encoded_jwt = SessionHandler().get_encoded_jwt(request.cookies['authorization'])
+    headers = {"jwt": encoded_jwt}
+    return headers
+
+
+def get_message(message_id, label, party_id):
+    logger.debug('Attempting to retrieve message', message_id=message_id, party_id=party_id)
+
+    headers = create_headers()
+    endpoint = 'secure-messaging/message'
+    parameters = {"message_id": message_id, "label": label, "party_id": party_id}
+    response = api_call('GET', endpoint, parameters=parameters, headers=headers)
+
+    if response.status_code != 200:
+        logger.error('Failed to retrieve message', message_id=message_id, party_id=party_id)
+        raise ApiError(response)
+
+    message = json.loads(response.text)
+    logger.debug('Retrieved message successfully', message_id=message_id, party_id=party_id)
```

Comment:
as aboove consistent with the other log message or are all debug or all info these kinf of.

Reply:
I think this was how it was when I got here, but I'll update it as I think they should all be info level anyway.